campari/campari_runner.py

```
@@ -0,0 +1,381 @@
+import numpy as np
+import pandas as pd
+import pathlib
+
+# Astronomy
+from astropy.io import fits
+import galsim
+
+# SN-PIT
+from snappl.sed import OU2024_Truth_SED, Flat_SED
+from snpit_utils.config import Config
+from snpit_utils.logger import SNLogger
+
+# Campari
+from campari.AllASPFuncs import (
+    add_truth_to_lc,
+    banner,
+    build_lightcurve,
+    build_lightcurve_sim,
+    extract_object_from_healpix,
+    find_all_exposures,
+    find_parquet,
+    get_object_info,
+    read_healpix_file,
+    run_one_object,
+    save_lightcurve,
+)
+
+
+class campari_lightcurve_model:
+    """This class holds the output of the Campari pipeline for a single SNID."""
+
+    def __init__(self, flux, sigma_flux, images, model_images, exposures, ra_grid, dec_grid, wgt_matrix,
+                 confusion_metric, best_fit_model_values, cutout_wcs_list, sim_lc):
+        """Initialize the Campari lightcurve model with the SNID and its properties."""
```

Add a Parmaeters section to the docstring that briefly defines what all the arguments are.

(Or, if you don't want to do it for this PR, create an issue telling somebody to do it.)

Also, see comment below where you actually make one of these.  Give everything other than self a default.  Either a sensible default, or just None.  That makes it clear that they this constructor is supposed to be called with kwargs, not with args.  (If you want to be really anal, you could throw an exception if any of the necessary ones are None.  But, you could also just count on the code crashing later if something is None that shouldn't be.)